iib/web/static/api_v1.yaml

```diff
@@ -183,6 +183,17 @@ paths:
               $ref: '#/components/schemas/RequestUpdate'
       security:
         - Kerberos Authentication: []
+  /healthcheck:
+    get:
+      description: Perform a health check on IIB by pinging this endpoint and awaiting expected return message
+      responses:
+        '200':
+          description: The health check OK message
+          content:
+            text/html:
```

Comment:
This isn't the right content type, is it?

Reply:
Fixed.